staging/cse/windows/provisioningscripts/windowsnodereset.ps1

```diff
@@ -82,19 +82,19 @@ Unregister-HNSRemediatorScriptTask
 # Stop services
 #
 Write-Log "Stopping kubeproxy service"
-Stop-Service kubeproxy
+Stop-Service kubeproxy -Force
 
 Write-Log "Stopping kubelet service"
-Stop-Service kubelet
+Stop-Service kubelet -Force
 
 if ($global:CsiProxyEnabled) {
     Write-Log "Stopping csi-proxy service"
-    Stop-Service csi-proxy
+    Stop-Service csi-proxy -Force
```

Comment:
Which services have unexpected dependency so we have to add `-force`? I think that we need to fix unexpected dependency instead of adding `-Force`

Reply:
Customers may add their own services, so if their service set dependency on `kubelet` and running, the `Stop-Service kubelet` would not stop successfully.